.pre-commit-config.yaml

```diff
@@ -11,6 +11,17 @@ repos:
   - id: end-of-file-fixer
   - id: trailing-whitespace
 
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.37.3
+  hooks:
+    - id: pyupgrade
+      args: ["--py38-plus"]
+
+- repo: https://github.com/fsouza/autoflake8
```

Comment:
The autoflake8 repo has been archived with instructions in the readme to use autoflake instead:

<img width="754" alt="image" src="https://user-images.githubusercontent.com/2895013/189509996-b7f9d7f1-5c3d-4f6b-80f6-2ebfb422af30.png">